ocean_provider/file_types/file_types.py

```diff
@@ -132,6 +133,8 @@ def __init__(
     def validate_dict(self) -> Tuple[bool, Any]:
         if not self.url:
             return False, "missing graphql endpoint"
+        if not self.query:
+            return False, "missing graphql query"
```

Comment:
can you please add this to the possible error documentation here: https://github.com/oceanprotocol/provider/blob/main/ocean_provider/routes/README.md#file-structure-errors

Reply:
added it myself to spare you some work :)